src/v4/matchImportsExports.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function matchImportsExports(node) {
```

Comment:
Is there any reason this is a JS file and not TS?

Reply:
It needs to be imported in a node script. We could rewrite those scripts in typescript but there aren't good types for jscodeshift and recast so it didn't seem worth it.

Reply:
TS would offer more consistency with the rest of the project and the other transforms, and would allow you to use things like async/await in Node 6. Additionally, the scripts can be run using ts-node